languages.toml

```
@@ -1355,3 +1355,13 @@ indent = { tab-width = 2, unit = "  " }
 [[grammar]]
 name = "meson"
 source = { git = "https://github.com/bearcove/tree-sitter-meson", rev = "feea83be9225842490066522ced2d13eb9cce0bd" }
+
+[[language]]
+name = "sshclientconfig"
+scope = "source.sshclientconfig"
+file-types = [".ssh/config", "/etc/ssh/ssh_config"]
+roots = []
+
+[[grammar]]
+name = "sshclientconfig"
+source = { git = "https://github.com/metio/tree-sitter-ssh-client-config.git", rev = "769d7a01a2e5493b4bb5a51096c6bf4be130b024" }
```

```suggestion
source = { git = "https://github.com/metio/tree-sitter-ssh-client-config", rev = "769d7a01a2e5493b4bb5a51096c6bf4be130b024" }
```

There are some places where we use this as a URL so we need to leave off the `.git`